packages/api-client/src/giphy/GiphyOptions.ts

```diff
@@ -17,6 +17,14 @@
  *
  */
 
+/** @see https://developers.giphy.com/docs/optional-settings/#rating */
+export enum GIPHY_RATING {
+  G = 'g',
```

Comment:
Let's keep the value of "g" but select a more descriptive key, i.e. `ALL_AGES_AND_PEOPLE`.